pkg/controller/controller.go

```diff
@@ -52,22 +53,29 @@ func AddToManager(m manager.Manager) error {
 		}
 	}
 	for _, f := range AddToManagerWithActuatorFuncs {
-		// Check if this is an AWS cluster and if not, add a dummy actuator:
+		// Check for supported platform types, dummy if not found:
 		// TODO: Use infrastructure type to determine this in future, it's not being populated yet:
 		// https://github.com/openshift/api/blob/master/config/v1/types_infrastructure.go#L11
 		var err error
 		var a actuator.Actuator
-		isAWS, err := isAWSCluster(m)
+		plat, err := platformType(m)
 		if err != nil {
 			log.Fatal(err)
 		}
-		if isAWS {
+		switch plat {
+		case configv1.AWSPlatformType:
 			log.Info("initializing AWS actuator")
 			a, err = awsactuator.NewAWSActuator(m.GetClient(), m.GetScheme())
 			if err != nil {
 				return err
 			}
-		} else {
+		case configv1.AzurePlatformType:
+			log.Info("initializing Azure actuator")
+			a, err = azure.NewActuator(m.GetClient())
+			if err != nil {
+				return err
+			}
+		default:
```

Comment:
maybe update the log message below to something like "initializing no-op actuator (not a supported platform)"